solidstart-tanstackquery-tailwind-modules/src/components/PRAndIssuesListItem/PRAndIssuesListItem.tsx

```diff
@@ -0,0 +1,122 @@
+// I must disable eslint because otherwise everytime I save the file
+// elsint brokes my class declaration at line #38 and the icon hasn't the right color
+
+/* eslint-disable */
```

Comment:
Can you just disable it for that line? 
`// eslint-disable-next-line`

Reply:
I solved the main issue so I removed it as well